docs/cookbook/using-S3-storage.md

```diff
@@ -0,0 +1,112 @@
+
+## Using FileBase for storage
+
+The FileBase storage service uses a truncated version of the S3 API. However, we can use this service in almost the same manner as the AWS S3 service. In this case, we must pass in one additional environment variable, `APOS_S3_ENDPOINT`, and **not** set the `APOS_S3_REGION` variable.
+
+1) Create a [FileBase account](https://filebase.com/). While the service does have a free tier, it doesn't allow public access. This means our site won't be able to load resources, so you need to select one of their paid plans.
+2) From the console switch to the "Buckets" tab and click on "Create Bucket".
+3) Give the bucket a name. This is the value we will pass to the `APOS_S3_BUCKET` variable.
+4) Leave the storage network as "IPFS" and click "Create Bucket".
+5) Set the bucket access to public using the toggle next to the bucket name.
+6) Click the "Access Keys" tab.
+7) This screen will display the key and secret values we will add to the `APOS_S3_KEY` and `APOS_S3_SECRET` variables.
+8) On this screen we can also get the value for the `APOS_S3_ENDPOINT` environment variable from the URL listed under 'S3 API Endpoint'. Note: you should **not** set the `APOS_S3_REGION` key.
+
+### FileBase Security
+While FileBase can accept the same type of CORS (cross-origin resource sharing) and ACL (access control list) rules as AWS S3, simply setting the security toggle to "public" on the "Buckets" screen should be enough for most sites.
+
+## Using Vultr for storage
+
+The Vultr storage service uses a truncated version of the S3 API. However, we can use this service in almost the same manner as the AWS S3 service. In this case, we must pass in one additional environment variable, `APOS_S3_ENDPOINT`, and **not** set the `APOS_S3_REGION` variable.
+
+1) Create a [Vultr account](https://www.vultr.com/). While the service doesn't have a free tier, you should be able to find a free credit offer if you are a new customer.
+2) Log in to your account and navigate to "Products".
+3) Select "Objects" to begin creating a new storage object.
+4) Select your preferred location. 
+5) Add a label for your object - this object can contain multiple storage buckets.
+6) Wait for the object to install and then click on the green arrow.
+7) Under the S3 Credentials you will see your Hostname, Secret Key, and Access Key. These will be used for the `APOS_S3_ENDPOINT`, `APOS_S3_SECRET`, and `APOS_S3_KEY` variables. Note: you should **not** set the `APOS_S3_REGION` key.
+8) Click the "Buckets" tab.
+9) Click the "Create Bucket" button.
+10) Add a bucket name following the noted limitations. This name will be used to set the `APOS_S3_BUCKET` variable.
+
+### Vultr Security
+The documentation says that uploaded files are private by default. However, I was able to access files without any security policy changes. You might need to set the `bucketObjectsACL` setting of your [`@apostrophecms/uploadfs` module]() to `public-read`.
```

Comment:
Yeah, AWS has hard blocks for the whole bucket over and above any ACLs you set on the files that have to be disabled... whereas Vultr just defaults individual files to private, which is not a problem for us because we explicitly set the ACL on each file the way we want it.

Reply:
You definitely don't need to do that, from the source:

```
      bucketObjectsACL = options.bucketObjectsACL || 'public-read';
```

So that is already the default value for `bucketObjectsACL` in uploadfs, to accommodate the fact that public files are the main use case (for uploadfs, if not S3).

Reply:
Dumb mistake. Changed and thanks.